x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/SearchableSnapshotIndexInput.java

```diff
@@ -48,6 +50,8 @@
  */
 public class SearchableSnapshotIndexInput extends BaseSearchableSnapshotIndexInput {
 
+    private static final Logger logger = LogManager.getLogger(SearchableSnapshotIndexInput.class);
```

Comment:
This isn't used? I mean it probably will be in future, but I think I'd prefer to add it when it's needed.

Reply:
:man_facepalming: Thanks, I removed it.